2-structured-data/test/_api-tests.js

```diff
@@ -13,57 +13,52 @@
 
 'use strict';
 
-const getRequest = require(`@google-cloud/nodejs-repo-tools`).getRequest;
-const test = require(`ava`);
+const getRequest = require('@google-cloud/nodejs-repo-tools').getRequest;
```

Comment:
copyright above still says 2017 though you did not touch that part.

Reply:
Hi Romola, 
As per review comment on this internal [PR](https://github.com/nareshqlogic/nodejs-spanner/pull/1 ) remaining unchanged Copyright year.
You don't need to update Copyrights, Leave it as it is. If this is a new file then you have to add Copyright with this:

/*
 * Copyright 2018 Google LLC. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
Notice this Inc is LLC now.